### PR TITLE
Create a reusable card component DetailCard

### PR DESCRIPTION
**Pull request by anurag-majumdar**
@divyamrastogi @blenderskool @realslimshanky @palnabarun @nikitasharma2692 @PrabhanshuAttri 

Created the custom reusable card component relating to:
https://github.com/UdacityMobileWebScholarship/showcase-app/issues/49

The checked in code contains 3 examples of the DetailCard component in App.js along with the reusable DetailCard component source code.

The image from my PC's screenshot:
https://user-images.githubusercontent.com/13219942/39900141-6880e1be-54dd-11e8-921b-5afb6bc0d6bc.png

Please review and let me know if any changes are required to be made. Thanks in advance.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,28 +1,43 @@
+@import url("https://fonts.googleapis.com/css?family=Nunito");
+body {
+  font-family: 'Nunito', sans-serif; }
+
 .App {
-  text-align: center;
-}
+  text-align: center; }
```
Comment (PrabhanshuAttri) [inline src/App.css]: Please follow [Google CSS Guide](https://google.github.io/styleguide/htmlcssguide.html#CSS_Formatting_Rules). The closing bracket should be in the next line.

If this is a generated file from App.scss file, could you please minify it.

Comment (anurag-majumdar) [reply src/App.css]: @PrabhanshuAttri This is being automatically set by the css watcher file from scss file actually when running npm start.

Comment (anurag-majumdar) [reply src/App.css]: @PrabhanshuAttri For this minification part, we can open another issue, that might help.

Comment (divyamrastogi) [reply src/App.css]: I've fixed this. Please take a pull from the upstream for the latest code.

Comment (anurag-majumdar) [reply src/App.css]: @divyamrastogi sure.

**Comment by anurag-majumdar** [comment]
The conflicts have been resolved. Please review my code and let me know your thoughts.

**Comment by anurag-majumdar** [comment]
A gentle reminder. Please review my commit and let me know if any changes are required to be made. 😃

**Comment by PrabhanshuAttri** [comment]
LGTM. I hope you were able to generate build without any errors.

**Comment by anurag-majumdar** [comment]
@PrabhanshuAttri yup. No issues at all.

**Comment by anurag-majumdar** [comment]
I am now making sure to remove css files from the commit and only keep scss files. 😃

**Comment by palnabarun** [comment]
@anurag-majumdar For your ease, you can add in your `.gitignore` the relevant directories of `.css` files. But, I see `.gitignore` that `**/*.css` is already there. Don't know how the `.css` is showing in your untracked files.

**Comment by anurag-majumdar** [comment]
@palnabarun currently it is having that config in gitignore. It is not tracking css files anymore. I checked. Let me know if this is good to go, then we can move on to the other dependencies related to this component. 😃

**Comment by palnabarun** [comment]
@anurag-majumdar That is why it is always suggested to rebase your changes to `upstream/master` whenever you start working on your changes so that you have the updated copy of code before working.

**Comment by palnabarun** [comment]
LGTM. Merging.

**Comment by anurag-majumdar** [comment]
@palnabarun I agree. Thanks for the heads up.

**Pull request merged**